tcms/testcases/migrations/0009_populate_missing_text_history.py

```diff
@@ -7,8 +7,8 @@ def forward_copy_data(apps, schema_editor):
 
     for test_case in test_case.objects.all():
         history = historical_test_case.objects.filter(
-                    case_id=test_case.pk
-                  ).order_by('-history_id').first()
+            case_id=test_case.pk
+            ).order_by('-history_id').first()
```

Comment:
I think line 11 should be 4 spaces to the left, on the same level as the history variable above but I don't see pylint complaining. You could also move everything to the line above.

Reply:
I think moving it all to line 10 works best here.